scalafmt-core/shared/src/main/scala/org/scalafmt/config/SortSettings.scala

```diff
@@ -27,7 +27,9 @@ object SortSettings {
     ModKey("protected", _.is[Mod.Protected]),
     //
     ModKey("lazy", _.is[Mod.Lazy]),
-    ModKey("open", _.is[Mod.Open])
+    ModKey("open", _.is[Mod.Open]),
+    ModKey("transparent", _.is[Mod.Transparent]),
+    ModKey("inline", _.is[Mod.Inline])
```

Comment:
I'd really put inline first. might it not replace `@inline` which is usually there?

Reply:
Hmm... do you mean we should move inline to the top of the list? Then we should have for example:
`inline override def foo = ???`
instead of:
`override inline def foo = ???`
?
I feel like the second one reads better though.